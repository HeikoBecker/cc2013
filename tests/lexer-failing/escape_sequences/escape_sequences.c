"\\\n\"\a\b\f\r\t\v\?\'"
'\\'
'\n'
'\"'
'\a'
'\b'
'\f'
'\r'
'\t'
'\v'
'\?'
'\''
'\0'
