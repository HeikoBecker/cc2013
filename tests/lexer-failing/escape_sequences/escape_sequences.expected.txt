escape_sequences.c:1:1: string-literal "\\\n\"\a\b\f\r\t\v\?\'"
escape_sequences.c:2:1: constant '\\'
escape_sequences.c:3:1: constant '\n'
escape_sequences.c:4:1: constant '\"'
escape_sequences.c:5:1: constant '\a'
escape_sequences.c:6:1: constant '\b'
escape_sequences.c:7:1: constant '\f'
escape_sequences.c:8:1: constant '\r'
escape_sequences.c:9:1: constant '\t'
escape_sequences.c:10:1: constant '\v'
escape_sequences.c:11:1: constant '\?'
escape_sequences.c:12:1: constant '\''
escape_sequences.c:13:1: constant '\0'
